classification_analysis.ipynb

```diff
@@ -1192,25 +1436,28 @@
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "37b50ace",
    "metadata": {
     "scrolled": false
    },
    "outputs": [],
    "source": [
-    "plot_cm('ios', [mcv_la, mcv_sj, mcv_ucb], 'clean', CIM)"
+    "plot_cm('ios', [mcv_la, mcv_sj, mcv_ucb], 'clean', CIM, criterion='distance')"
```

Comment:
why are we changing this notebook to only work with distance?
Again, the point of having the criterion passed in is so that we can compare `distance` and `duration`

I don't see a single duration calculation in the notebook at this point.

Reply:
Understood. I will add the duration calculations in subsequent cells as well.